apps/firehose_to_es_processor/lib/firehose_record.py

```diff
@@ -4,6 +4,15 @@
 import base64
 from datetime import datetime
 from lib.util import extract_json
+import os
+from airbrake.notifier import Airbrake
+import re
+
+airbrake_flag = os.environ.get('AIRBRAKE_FLAG')
+airbrake_notifier = None
+if airbrake_flag and airbrake_flag == "True":
+    airbrake_notifier = Airbrake(project_id=os.environ.get("AIRBRAKE_PROJECT_ID"), api_key=os.environ.get("AIRBRAKE_API_KEY"))
```

Comment:
Are there cases in which these environment variables are not required? If not, perhaps it's better to just have a `os.environ[<key>]` and fail faster.